src/freenas-installer/scale/build_archive.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding=utf-8 -*-
+import argparse
+import contextlib
+import glob
+import json
+import logging
+import os
+import shutil
+import subprocess
+import tempfile
+import textwrap
+
+logger = logging.getLogger(__name__)
+
+if __name__ == "__main__":
+    if os.getuid() != 0:
+        raise RuntimeError("This script must be ran by root")
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--skip-rootfs", action="store_true")
+    parser.add_argument("version")
+    args = parser.parse_args()
+
+    os.chdir(os.path.normpath(os.path.dirname(__file__)))
+
+    if not args.skip_rootfs:
+        with contextlib.suppress(FileNotFoundError):
+            os.unlink("/usr/share/debootstrap/scripts/truenas")
+        subprocess.run("ln -s /usr/share/debootstrap/scripts/bullseye /usr/share/debootstrap/scripts/truenas",
```

Comment:
@themylogin 

Looks like you still have lots of code for doing debootstrap and similar steps in here. Did you intend to use the 'make update' functionality of truenas-build where we already bootstrap the rootfs image, install packages, etc?